### ISSUE TITLE
HasMany & PredictiveInput components need a11y pass

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2293]
- Have improper use of `<nav>` element.
- Both fail to provide valuable information to screen reader, including descriptions of the form fields, feedback for predictive inputs, and updates when tags and subjects are added/removed.
